test/integration/model/sync.test.js

```diff
@@ -215,16 +215,12 @@ describe(Support.getTestDialectTeaser('Model'), () => {
           const results = await this.sequelize.getQueryInterface().showIndex(User.getTableName());
           if (dialect === 'sqlite') {
             // SQLite doesn't treat primary key as index
-            // However it does create an extra "autoindex", except primary == false
```

Comment:
Why does that change?

Reply:
The PR that introduced this bug added this to the test to make it pass https://github.com/sequelize/sequelize/pull/12420/files#diff-81b952d2bbfce09ba3fc0712ea8248f597d3d3421d819a1c69eb06f68fcf318eR218

Reply:
This test still needs to change with my fix. I don't think it should have been changed in #12420.